webshare/share-empty-manual.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Manual Test: Share with empty ShareData</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
+
+        setupManualShareTest(false, '', '', '');
+        callWhenButtonClicked(() => navigator.share({}));
```

Comment:
Reported a spec bug [here](https://github.com/WICG/web-share/issues/43) and a spec fix is incoming.

Added a test to share-simple-manual that it resolves with undefined. I don't think it's worth testing for this in every case.